A71-xds-fallback.md

```diff
@@ -71,19 +71,19 @@ control plane, which is not desirable.
 gRPC servers using the xDS configuration will share the same XdsClient instance
 keyed with a dedicated well-known key value.
 
-Changes XdsClient to per-data plane target would enable gRPC to switch to
+Changing the XdsClient to per-data plane target enables gRPC to switch to
```

Comment:
This language still assumes somewhere else we are saying we are making this change. "Implementations will change the XdsClient to be per-data plane target, which enables gRPC to switch..." or "XdsClient will be changed to be per-data plane target, which enables..." (which could be misinterpreted, but you use similar language elsewhere so it should be fine) or similar.

Reply:
Updated the text. Please take another look.

Reply:
I don't see any new changes.

Reply:
See here: http://localhost:3000 😉 

(I pushed the commit I forgot to push)